rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -491,7 +491,8 @@ static std::string get_node_user_data(const char * node_name, const char * node_
 extern "C" rmw_node_t * rmw_create_node(
   rmw_context_t * context, const char * name,
   const char * namespace_, size_t domain_id,
-  const rmw_node_security_options_t * security_options)
+  const rmw_node_security_options_t * security_options,
+  bool localhost_only)
```

Comment:
The implementation block should contain `(void)localhost_only` to avoid a compiler warning.